### PR TITLE
[SP-7394] Move campaignEnv to the top level request body

### PR DESCRIPTION
**Pull request by vilasma**

### CODE REVIEW

File: ConsentViewController/Classes/SPConsentManager.swift [hunk 1]
```diff
@@ -46,20 +46,22 @@ import Foundation
         }
     }
 
-    public convenience init(accountId: Int, propertyName: SPPropertyName, campaigns: SPCampaigns, delegate: SPDelegate?) {
+    public convenience init(accountId: Int, propertyName: SPPropertyName, campaignEnv: SPCampaignEnv = .Public, campaigns: SPCampaigns, delegate: SPDelegate?) {
```
Comment (andresilveirah) [inline ConsentViewController/Classes/SPConsentManager.swift:49]: @vilasma let's rename the parameter from `campaignEnv` to `campaignsEnv`. I think this makes it a bit more explicit this parameter refers to all campaigns.

File: ConsentViewController/Classes/SPConsentManager.swift [hunk 2]
```diff
@@ -46,20 +46,22 @@ import Foundation
...
-            spClient: SourcePointClient(accountId: accountId, propertyName: propertyName, timeout: SPConsentManager.DefaultTimeout),
+            spClient: SourcePointClient(accountId: accountId, propertyName: propertyName, campaignEnv: campaignEnv, timeout: SPConsentManager.DefaultTimeout),
             storage: SPUserDefaults(storage: UserDefaults.standard)
         )
     }
 
     init(
         accountId: Int,
         propertyName: SPPropertyName,
+        campaignEnv: SPCampaignEnv,
```
Comment (andresilveirah) [inline ConsentViewController/Classes/SPConsentManager.swift:64]: Same here -> `campaignsEnv`

**Review by andresilveirah — COMMENTED** [review]
Apart from the minor renaming 💯

File: ConsentViewController/Classes/SPConsentManager.swift (hunk 1 above)
Comment (vilasma) [reply ConsentViewController/Classes/SPConsentManager.swift]: Done

File: ConsentViewController/Classes/SPConsentManager.swift (hunk 2 above)
Comment (vilasma) [reply ConsentViewController/Classes/SPConsentManager.swift]: Done

**Pull request merged**